provider/kv.go

```diff
@@ -112,6 +114,23 @@ func (provider *Kv) get(keys ...string) string {
 	return string(keyPair.Value)
 }
 
+func (provider *Kv) getBool(keys ...string) bool {
```

Comment:
This is gonna feel dumb but, why not doing something like that (to avoid duplicate code ?) :

```go
func (provider *Kv) getBool(keys ...string) bool {
	value := get(keys...)
	b, err := strconv.ParseBool(string(keyPair.Value))
	if err != nil {
		log.Error("Error getting key: ", joinedKeys, err)
		return false
	}
	return b
}
```